modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm

```diff
@@ -707,49 +755,52 @@
+		if(25 to 50)
+			throw_arousalalert("arousal_medium", arousal_alert, target_human)
+		if(50 to 75)
+			throw_arousalalert("arousal_high", arousal_alert, target_human)
+		if(75 to INFINITY) //to prevent that 101 arousal that can make icon disappear or something.
+			throw_arousalalert("arousal_max", arousal_alert, target_human)
+
+	if(target_human.arousal > 1)
+		switch(target_human.pain)
+			if(-100 to 1) //to prevent same thing with pain
+				arousal_alert?.cut_overlay(arousal_alert.pain_overlay)
 			if(1 to 25)
-				throw_arousalalert("arousal_small", arousal_alert, target_human)
+				overlay_pain("small", arousal_alert)
 			if(25 to 50)
-				throw_arousalalert("arousal_medium", arousal_alert, target_human)
+				overlay_pain("medium", arousal_alert)
 			if(50 to 75)
-				throw_arousalalert("arousal_high", arousal_alert, target_human)
-			if(75 to INFINITY) //to prevent that 101 arousal that can make icon disappear or something.
-				throw_arousalalert("arousal_max", arousal_alert, target_human)
-
-		if(target_human.arousal > 1)
-			switch(target_human.pain)
-				if(-100 to 1) //to prevent same thing with pain
-					arousal_alert?.cut_overlay(arousal_alert.pain_overlay)
-				if(1 to 25)
-					overlay_pain("small", arousal_alert)
-				if(25 to 50)
-					overlay_pain("medium", arousal_alert)
-				if(50 to 75)
-					overlay_pain("high", arousal_alert)
-				if(75 to INFINITY)
-					overlay_pain("max", arousal_alert)
-
-		if(target_human.arousal > 1)
-			switch(target_human.pleasure)
-				if(-100 to 1) //to prevent same thing with pleasure
-					arousal_alert?.cut_overlay(arousal_alert.pleasure_overlay)
-				if(1 to 25)
-					overlay_pleasure("small", arousal_alert)
-				if(25 to 60)
-					overlay_pleasure("medium", arousal_alert)
-				if(60 to 85)
-					overlay_pleasure("high", arousal_alert)
-				if(85 to INFINITY)
-					overlay_pleasure("max", arousal_alert)
-		else
-			if(arousal_alert?.pleasure_level in list("small", "medium", "high", "max"))
-				arousal_alert.cut_overlay(arousal_alert.pleasure_overlay)
-			if(arousal_alert?.pain_level in list("small", "medium", "high", "max"))
-				arousal_alert.cut_overlay(arousal_alert.pain_overlay)
+				overlay_pain("high", arousal_alert)
+			if(75 to INFINITY)
+				overlay_pain("max", arousal_alert)
+
+	if(target_human.arousal > 1)
+		switch(target_human.pleasure)
+			if(-100 to 1) //to prevent same thing with pleasure
+				arousal_alert?.cut_overlay(arousal_alert.pleasure_overlay)
+			if(1 to 25)
+				overlay_pleasure("small", arousal_alert)
+			if(25 to 60)
+				overlay_pleasure("medium", arousal_alert)
+			if(60 to 85)
+				overlay_pleasure("high", arousal_alert)
+			if(85 to INFINITY)
+				overlay_pleasure("max", arousal_alert)
+	else
+		if(arousal_alert?.pleasure_level in list("small", "medium", "high", "max"))
+			arousal_alert.cut_overlay(arousal_alert.pleasure_overlay)
+		if(arousal_alert?.pain_level in list("small", "medium", "high", "max"))
```

Comment:
Those two lists should be static, so we don't have to instantiate them every time for nothing.